rs/nns/governance/src/network_economics.rs

```diff
@@ -29,6 +29,232 @@ impl NetworkEconomics {
             voting_power_economics: Some(VotingPowerEconomics::with_default_values()),
         }
     }
+
+    /// This verifies the following:
+    ///
+    ///     1. max_proposals_to_keep_per_topic > 0. The problem with 0 is that
+    ///        all future proposals would be blocked. Of course, in practice,
+    ///        this would never occur, because ManageNetworkEconomics does not
+    ///        have the ability to set this field to 0, and it already has
+    ///        positive value.
+    ///
+    ///     2. neurons_fund_economics and voting_power_economics are
+    ///
+    ///         i.  set. In practice, we would not encounter None here, for
+    ///             reasons similar to why we would not see
+    ///             max_proposals_to_keep_per_topic being set to 0.
+    ///
+    ///         ii. valid, according to their types. See their respective
```

Comment:
Thanks for the nice doc comment!

Would be awesome to code-link to the respective validation function to aid IDE navigation